### ISSUE TITLE
bug: [Bug] Ion list header inside card wrong margin

### ISSUE THREAD

**Issue by luchillo17** [ionic-team/ionic#7211]
**Type**: <span ionic-type>bug</span>

**Ionic Version**: <span ionic-version>2.x</span>

**Platform**: <span ionic-platform>all</span>  

<span ionic-description>
The margin top of `ion-list-header` elements shouldn't be negative for non first card lists.

![image](https://cloud.githubusercontent.com/assets/9698639/16601314/64d3b41c-42d1-11e6-8446-ea7314edb769.png)

![image](https://cloud.githubusercontent.com/assets/9698639/16601322/78eed774-42d1-11e6-8105-d2f7fc101467.png)

Basically the 16px under the first list doesn't exist for lists inside cards, meanwhile the -16px in the next lists header are always there thus causing this kind of graphic bug where the orange header is too close to the last item of the first list:

![image](https://cloud.githubusercontent.com/assets/9698639/16601374/c964c880-42d1-11e6-8b9a-2b541399010e.png)

</span>

<span is-issue-template></span>

**Comment by jgw96** [comment]
Hello, thanks for opening an issue with us! Would you be able to provide a plunker that demonstrates this issue? Thanks for using Ionic!

**Comment by Luchillo** [comment]
Ok, it's very simple, have you a base plunker to start?

**Comment by jgw96** [comment]
@Luchillo yep, here you go! Thanks!

**Comment by Luchillo** [comment]
Here you go? i don't see any links to it.

**Comment by jgw96** [comment]
Ahh sorry, seems like github did not take the link. http://plnkr.co/edit/me3Uk0GKWVRhZWU0usad?p=preview

**Comment by Luchillo** [comment]
Here you go, [card list header style error](http://plnkr.co/edit/asI9mV?p=preview) check the header of 2º list in the card in the home page, it's way too close, with current styles it doesn't look too out of place but with mine that have borders you can see almost overlapping.
